web/src/components/tasks/business-formation/name/DbaNameSearch.tsx

```diff
@@ -7,51 +7,32 @@ import { useConfig } from "@/lib/data-hooks/useConfig";
 import { useUserData } from "@/lib/data-hooks/useUserData";
 import analytics from "@/lib/utils/analytics";
 import { useMountEffect } from "@/lib/utils/helpers";
-import { NameAvailability } from "@businessnjgovnavigator/shared/";
 import { ReactElement, useContext } from "react";
 
 export const DbaNameSearch = (): ReactElement => {
   const { Config } = useConfig();
-  const { updateQueue } = useUserData();
-  const { setBusinessNameAvailability } = useContext(BusinessFormationContext);
+  const { state } = useContext(BusinessFormationContext);
+  const { updateQueue: userData } = useUserData();
```

Comment:
I don't understand this syntax, actually. What is this line doing with the colon?

Reply:
I see... below, we're using `userData?.current()` instead of `updateQueue.current()`

Why? This seems like a really confusing pattern for other developers, in terms of breaking expectations of what people expect when they see a variable named `userData`

Reply:
I don't remember my exact motivation. Changing that back to `updateQueue`.